zkevm-circuits/src/rlp_circuit_fsm.rs

```diff
@@ -1121,6 +1148,14 @@ impl<F: Field> RlpCircuitConfig<F> {
             // Bytes => DecodeTagStart
             cb.condition(tidx_eq_tlen, |cb| {
                 // assertions
+                let (lt, eq) = tlength_lte_mlength.expr(meta, Some(Rotation::cur()));
+                cb.require_equal(
+                    "tag_length <= max_length",
+                    sum::expr([
```

Comment:
nitpick: I would personally prefer `or` over `sum` as its more intuitive in this case

Reply:
But `or` has higher degree than `sum`. How about I leave some comments there?

Reply:
Okay, it works in this case as both `lt` and `eq` are boolean, so `sum` and `or` will actually give us the same result. But yes, a comment to explain it would be better as `sum` by itself shouldn't be used (but it works in this case).